apps/omg_eth/test/support/dev_helpers.ex

```diff
@@ -157,4 +123,59 @@ defmodule OMG.Eth.DevHelpers do
 
     fn -> WaitFor.repeat_until_ok(f) end |> Task.async() |> Task.await(timeout)
   end
+
+  # private
```

Comment:
I thought this comment means all methods below would be private. There are two `def` instead of `defp` here.